openedx_tagging/core/tagging/models/system_defined.py

```diff
@@ -75,10 +76,10 @@ def get_instance(self) -> models.Model | None:
         instance_id = self.tag.external_id if self.tag else None
         if instance_id:
             try:
-                return self.tag_class_model.objects.get(pk=instance_id)
+                return self.tag_class_model.objects.get(pk=instance_id)  # type: ignore[attr-defined]
```

Comment:
```suggestion
               # See https://github.com/typeddjango/django-stubs/issues/1684 for why we need to ignore this.
                return self.tag_class_model.objects.get(pk=instance_id)  # type: ignore[attr-defined]
```

It's good to add the comment here next to the code so it's easier to find in the future.